src/util/jira.ts

```diff
@@ -1,13 +1,14 @@
 import { JiraAuthor } from "../interfaces/jira";
+import _ from "lodash";
 
 export const getJiraAuthor = (...authors: (Author | undefined)[]): JiraAuthor => {
 	const author = Object.assign({}, ...authors);
-	return author.login || author.name ? {
-		avatar: author.avatar_url || author.avatarUrl || `https://github.com/users/${author.login}.png`,
+	return author.login || author.name ? _.pickBy({
+		avatar: author.avatar_url || author.avatarUrl || (author.login ? `https://github.com/users/${author.login}.png` : undefined),
```

Comment:
Should there be only one `author.avatar_url` ?

Reply:
Nope.  Depends if the author object comes from the REST API (avatar_url) or the GraphQL API (avatarUrl).  Fun right?